mlflow/types/llm.py

```diff
@@ -101,8 +101,6 @@ class ChatParams(_BaseDataclass):
             comprising the top 10% probability mass are considered.
         top_k (int): An optional param for reducing the vocabulary size to top k tokens
             (sorted in descending order by their probabilites).
-        logit_bias (dict): An optional param for specifying a dict that maps tokens
-            (specified by their token ID in the tokenizer) to an associated bias value.
```

Comment:
`logit_bias` requires type of `Map[ int -> int ]` type which MLflow hasn't support yet.
So I remove it for now.

`logit_bias` is also a less common param so supporting it is not urgent to